java-test-samples/apigw-lambda-ddb/src/test/java/com/example/TicketEnd2EndTest.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: MIT-0
+ */
+
+package com.example;
+
+import org.junit.Assert;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.cloudformation.CloudFormationClient;
+import software.amazon.awssdk.services.cloudformation.model.DescribeStacksRequest;
+import software.amazon.awssdk.services.cloudformation.model.DescribeStacksResponse;
+import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+
+// This test requires an AWS account and assumes the SAM template part of this repo is deployed to AWS
+// This test runs END 2 END by HTTP request to API Gateway URL and
+// verifies if entries are persisted in DDB
+public class TicketEnd2EndTest {
+  //Name of the stack you used when deploying SAM template
+  private static final String STACK_NAME = "APIGW-Lambda-DDB-Sample";
+  private static DynamoDbClient ddbClient;
+  private static CloudFormationClient cloudFormationClient;
+  private static String endPoint = "";
+  private List<String> ticketList = new ArrayList<>();
+
+  @BeforeAll
+  static void setUp() {
+    ddbClient = DynamoDbClient.builder()
+      .region(Region.US_EAST_1)
+      .build();
+
+    cloudFormationClient = CloudFormationClient
+      .builder().region(Region.US_EAST_1).build();
+
+    DescribeStacksResponse response = cloudFormationClient.describeStacks(DescribeStacksRequest.builder()
+      .stackName(STACK_NAME)
+      .build());
+    endPoint = response.stacks().get(0).outputs().get(0).outputValue();
```

Comment:
```suggestion
    apiEndPoint = response.stacks().get(0).outputs().get(0).outputValue();
```
Also not sure how we can make it more clear than get(0)...get(0)? Can't we retrieve by name?

Reply:
Fixed